combat-trainer.lic

```diff
@@ -2880,24 +2916,13 @@ class AttackProcess
         waitrt?
       end
     else
-      dual_load = game_state.dual_load?
-
-      if dual_load
-        # Occasionally, there may be no game output after loading.
-        # To be on the safe side, we do two attempts to get output.
+      if game_state.dual_load?
         fput('load arrows')
-        case bput('load arrows', 'You reach into', 'You carefully load', 'already loaded', 'in your hand', "As you try to reach", "You don't have the proper ammunition", 'Such a feat would be impossible without the winds to guide', 'but are unable to draw upon its majesty', 'without steadier hands', 'Your .* makes the task more difficult')
-        when "As you try to reach", "You don't have the proper ammunition", 'Such a feat would be impossible without the winds to guide', 'but are unable to draw upon its majesty', 'without steadier hands'
-          dual_load = false # fallback to single load
-          game_state.loaded = false
-        else
-          game_state.loaded = true
+        pause 0.5 until (load_return = bput('load arrows', 'You reach into', 'already loaded', 'in your hand', 'Such a feat would be impossible without the winds to guide', 'but are unable to draw upon its majesty', 'without steadier hands', 'Your .* makes the task more difficult')) != ''
+        if ['but are unable to draw upon its majesty', 'without steadier hands', 'Such a feat would be impossible without the winds to guide'].include?(load_return)
+          pause 0.5 until bput('load', 'You reach into', 'You carefully load', 'already loaded', 'in your hand') != ''
         end
-      end
-
-      unless dual_load
-        # Occasionally, there may be no game output after loading.
-        # To be on the safe side, we do two attempts to get output.
+      else
         fput('load')
         case bput('load', 'You reach into', 'You carefully load', 'already loaded', 'in your hand', "As you try to reach", "You don't have the proper ammunition")
         when "As you try to reach", "You don't have the proper ammunition"
```

Comment:
⚡ Merge conflict, this is inadvertently reverting changes introduced in https://github.com/rpherbig/dr-scripts/pull/4686 that should be kept